server/migrations/20201031091221-service-reword-status.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('t_service', 'status', {
```

Comment:
This will drop the table, same problem as removeColumn

Reply:
:'( we have to live with the type for ever?

Reply:
We have at least to wait that Sequelize improve the way they do a change/remove/rename column! I'm looking forward to their v6 which is in beta. 

But for now, those methods are not authorized in Gladys. I need to add another check for this function too

Reply:
I add changeColumn restriction.

Reply:
And keep workaround.

Reply:
Thanks for the changes!